src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
```

Comment:
Or, let me ask something else: why move this rule inside `.tab`, instead of having it in `.tab.deselected` like before?